### ISSUE TITLE
Split matrix-doc into multiple repositories

### ISSUE THREAD

**Issue by non-Jedi** [matrix-org/matrix-doc#1037]
The current structure of matrix-doc is extremely confusing to me, and it was
even more confusing to me than it currently is when I first encountered it.
At minimum, to encourage community contributions to matrix-doc, it should be
split into two repositories. I personally believe it should be split into at
least 3:

1. matrix-spec. If the spec was in a separate repo, it would be much clearer
   what needs to be editted to make spec changes. I would personally prefer
   that the client-server spec, server-server spec, and AS spec each had
   their own separate repo, but this is a good first step.
2. matrix-web. This is for content other than the spec that goes on the
   website (right now mostly projects).
3. matrix-doc. This is for everything else. Without having the spec and
   content for the website here, we could also allow community members who
   don't want to use google docs to make spec proposals in this repo.
   
At the very least, I think the spec urgently needs to be broken out of
matrix-doc.

**Comment by non-Jedi** [comment]
I'd like to reiterate how important I think it is to have a place that
community members can contribute informal spec work. I for one am extremely
uncomfortable even commenting on the existing spec docs via google's system
for various reasons having to do with google itself, and I know I'm not the
only one.

Github isn't perfect, but I think it's better suited for this kind of thing
because of data portability. It also gives the community an easy, formal way
to ask for their stuff to be included. To be clear, I'm not asking that the
matrix core team shifts their existing informal spec work to a git repo,
just that such a repo is available for the community to PR against.

**Comment by richvdh** [comment]
@non-Jedi: Thank you for the feedback.

Splitting up this repo certainly sounds like a good idea. Certainly the supporting docs could usefully be split out. To be honest I think there is only "spec" and "supporting docs".

The main problem I see with this is actually the issues: there are a bunch of issues and pull-requests open against this repo, most of which relate to the spec. Github doesn't provide a clean way to move issues from one repo to another. Having the issues in a separate repo to the spec sounds horrible, having them split across two sounds worse. So, we may have to accept that the spec is going to stay in this repo (though we could rename it), and other things can move out.

In short, I'll look into moving the supporting docs elsewhere.

**Comment by richvdh** [comment]
The supporting-docs are elsewhere as of https://github.com/matrix-org/matrix-doc/commit/e7772af5c32e094d849c2ca219295a20abb5c22e, and I've cleaned a couple of other things out of the project root too, so I'm considering this done.

**Comment by non-Jedi** [comment]
Thanks @richvdh! This is awesome. For me at least, makes it a lot easier to
tell what's going on with the spec. Would prefer if "drafts" and "attic"
were elsewhere as well, but I see how they don't really fit into the new
"matrix.org" repo. Anyways, this is awesome, and thanks again.
